content/syllabuses/java-bridging-course.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Java Bridging Course 
+ready: True
+---
+
+
+Bridging Course
+
+You’ll notice that the content below is divided up into 2 major sections. Soft skills and Hard skills. 
+
+## Soft Skills 
+
+### 1. Learning how to learn: 
+{{% contentlink "topics/bridging-course-soft-skills/learning-how-to-learn" %}}
+
+### 2. Communication in a workplace
+{{% contentlink "topics/bridging-course-soft-skills/communication" %}}
+
+### 3. Emotional Intelligence
+{{% contentlink "topics/bridging-course-soft-skills/emotional-intelligence" %}}
+
+### 4. Cognitive biases
+{{% contentlink "topics/bridging-course-soft-skills/cognitive-biases" %}}
+
+### 5. Developer habits
+{{% contentlink "topics/bridging-course-soft-skills/developer-habits" %}}
+
+
+## Hard Skills 
+
+### 1. Maths for programmers
+{{% contentlink "topics/bridging-course-hard-skills/maths-for-programmers" %}}
+
+### 2. Solo Learn 
+{{% contentlink "topics/bridging-course-hard-skills/maths-for-programmers" %}}
```

Comment:
Shouldn't all these Hard Skills link to different content? They all link to maths-for-programmers

Reply:
I'm also not convinced that these headings are useful. The content links get rendered using the titles of the content. So this page will look quite repetitive.

Run hugo and look at the page. If it looks repetitive then clean it up